Tasks/google-play-release/Tests/L0.ts

```diff
@@ -70,6 +70,28 @@ describe('L0 Suite google-play-release', function () {
         done();
     });
 
+    it('test found obb file', (done) => {
+        const testFile = path.join(__dirname, 'L0ObbFileFound.js');
+        const testRunner = new ttm.MockTestRunner(testFile);
+
+        testRunner.run();
+
+        assert(testRunner.stdOutContained('Found Obb file for upload: /path/to/obbfolder/filename.obb'), 'Did not print the expected message: ' + JSON.stringify(testRunner));
+        assert(testRunner.succeeded, 'task should have succeeded' + JSON.stringify(testRunner));
```

Comment:
nit: you should probably remove the ' + JSON.stringify(testRunner)' from these lines.